UI/src/app/chart/chart.component.ts

```diff
@@ -1,8 +1,7 @@
-import { Component, Input, OnInit } from '@angular/core';
-import * as Plotly from 'plotly.js/lib/core';
+import { Component, OnInit } from '@angular/core';
+import * as Plotly from 'plotly.js';
```

Comment:
Changing that line caused the warning. Maybe we can add both statements if you need this one for sure.